pyiron_contrib/continuum/fenics.py

```diff
@@ -195,8 +195,13 @@ def run_static(self):
         Solve a PDE based on 'LHS=RHS' using u and v as trial and test function respectively. Here, u is the desired
         unknown and RHS is the known part.
         """
+        self.status.running = True
         self.u = FEN.Function(self.V)
         FEN.solve(self.LHS == self.RHS, self.u, self.BC)
+        self.collect_output()
```

Comment:
The `collect_output` is triggered by `run()` if `self.status.collect`, thus
```
self.status.collect = True
self.run()
```

Reply:
Yes, this is true. I did it the other way because it felt more explicit, but you're absolutely right that the way you suggest is more consistent with the code base. I fixed it.

However, this brings me back again to the point that we need to revisit our pyiron_base architecture here -- these canonical methods should all be wrapped with something that automatically updates the status so that we don't duplicate, e.g. `self.status.collect = True; self.run()` in (almost) every subclass of `GenericJob`!!!